python/generate/templates/api.mustache

```diff
@@ -251,12 +251,12 @@ class {{classname}}(object):
         # HTTP header `Content-Type`
         header_params['Content-Type'] = self.api_client.select_header_content_type(  # noqa: E501
             [{{#consumes}}'{{{mediaType}}}'{{^-last}}, {{/-last}}{{/consumes}}])  # noqa: E501
+        {{/hasConsumes}}
```

Comment:
Just for my own understanding, what does this /hasConsumes control?

Reply:
It sets the content type header if specified. The issue we had was that we incorrectly/accidently set the LUSID headers inside this block which meant that apis that didn't specify a content type didn't set the LUSID header